vs-code/vscode.md

```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
 
 [Slides: Local Development with VS Code](https://docs.google.com/presentation/d/1QAMxrS1ZOvtSn7MuYxTmeanrMQi_O8Z5UVq-4LzLqso/edit?usp=sharing)
 
+### VSCode Tips
+
+- **Auto Indent** - VS Code package to format your JavaScript / TypeScript / CSS using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). Or Select the text you want to automatically indent. Click Format Selection in Edit, Advanced, or press **CTRL+K, CTRL+F**. 
+
+- **Indent multiple lines** - Just highlight the lines you want, then press tab to indent the entire code block, or **shift+tab** to un-indent the entire code block.
+
+- **Code Folding** - Folds the innermost uncollapsed region at the cursor.<br>
+                     **"Ctrl + Shift + ["** on Windows and Linux.<br>
+                     **"⌥ + ⌘ + ["** on macOS.
+                     
+- **Autosave** - The easiest way to turn on **Auto Save** is with the menu **File > Auto Save** toggle that turns on and off save after                  a delay.                    
+
+- **Key-Value** - Visual Studio Code lets you perform most tasks directly from the keyboard.
```

Comment:
I don’t think titling this section “key-value” helps learners understand what those terms mean. Choose a title more about how this fits into vscode, and add a section on what a key-value pair is. Apprentices will do this in week 1, about a month before they learn about JS objects.

Reply:
ok